awx/main/analytics/collectors.py

```diff
@@ -248,7 +248,12 @@ def _copy_table(table, query, path):
                               main_jobevent.role, 
                               main_jobevent.job_id, 
                               main_jobevent.host_id, 
-                              main_jobevent.host_name
+                              main_jobevent.host_name,
+                              CAST(main_jobevent.event_data::json->>'start' AS TIMESTAMP) AS start,
```

Comment:
```sql
awx-dev=> SELECT CAST(main_jobevent.event_data::json->>'start' AS TIMESTAMP) AS start FROM main_jobevent LIMIT 10;
           start
----------------------------




 2020-04-08 16:46:36.19182


 2020-04-08 16:46:38.339398


(10 rows)
```

Reply:
maybe `timestamp with time zone`, or is this always utc?

Reply:
It's missing the timezone when it comes from runner. We could change that, but... meh

Reply:
ok